src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -53,6 +54,7 @@ public RecheckImpl( final String suiteName ) {
 	}
 
 	public RecheckImpl( final FileNamerStrategy fileNamerStrategy, final String suiteName ) {
+		ProjectConfiguration.getInstance().initializeConfiguration();
```

Comment:
Maybe this should go into `ensureConfigurationInitialized()` or even `Configuration#ensureLoaded()`?